turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -143,6 +143,22 @@ use crate::{
     TreeShakingMode,
 };
 
+#[turbo_tasks::value(shared)]
+#[derive(Clone, Debug, Default)]
+pub struct DeclaredCjsGlobals {
+    pub dirname: bool,
+    pub filename: bool,
+    pub require: bool,
```

Comment:
Actually, we currently only need the `require` bit.
Previously, `__dirname` was always included, but I don't know why that was done. Seems to work fine when excluding it on ESM
https://github.com/vercel/next.js/blob/ea1618d3fdd25d255650bd2349d0fde11ab5637c/turbopack/crates/turbopack-ecmascript/src/chunk/item.rs#L99-L100